examples/vue-composition-api/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "vue-project",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check \"build-only {@}\" --",
+    "preview": "vite preview",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --build"
+  },
+  "dependencies": {
+    "pspdfkit": "^2024.8.1",
```

Can we use the new package instead, please?:

```suggestion
    "@nutrient-sdk/viewer": "^1.0.0",
```